includes/admin/class-sensei-setup-wizard.php

```diff
@@ -417,17 +417,51 @@ public function get_mailing_list_form_data() {
 		];
 	}
 
+	/**
+	 * Get data used for WooCommerce.com purchase redirect for feature installation.
+	 *
+	 * @return array The data.
+	 */
+	public function get_woocommerce_connect_data() {
+
+		$is_woocommerce_installed = Sensei_Utils::is_plugin_present_and_activated( 'Woocommerce', 'woocommerce/woocommerce.php' );
+
+		if ( ! $is_woocommerce_installed ) {
+			return [];
+		}
+
+		$back_admin_path = add_query_arg(
+			[
+				'page'     => $this->page_slug,
+				'step'     => 'features',
+				'feedback' => 1,
+			],
+			'admin.php'
+		);
+
+		return [
```

Comment:
Do you think it'd be worth doing a method check and pulling these directly from `\WC_Admin_Addons::get_in_app_purchase_url_params`? I guess it is a gamble of what is most likely to break/change: the method name (hopefully deprecated) or their implementation for what they pass to WCCOM.

Reply:
Nice find, added a branch for this in 7430ab1. The previous code is still there as a fallback so we can prepare the parameters in case WooCommerce is not yet installed.